linux/net/rina/dtcp-utils.c

```diff
@@ -346,6 +349,14 @@ static struct dtcp_config * dtcp_config_create_gfp(gfp_t flags)
           }
         */
 
+        tmp->dtcp_ps = policy_create();
```

Comment:
Same as above. The general question is: what's the point in storing these "struct policy" object? They seem to be allocated on create, deallocated on destroy and never used. Why don't we just purge away this code?